runtime/CSharp/Antlr4BuildTasks/Antlr4.CodeGenerator.targets

```diff
@@ -196,4 +196,12 @@
   </Choose>
 
   <Import Condition="'$(Antlr4IsSdkProject)' == 'True'" Project="Antlr4.CodeGenerator.DefaultItems.targets" />
+  
+  <ItemGroup Condition="'$(NCrunch)' == '1'">
+    <None Include="$(MSBuildThisFileDirectory)..\tools\**\*" />
+    <None Include="@(Antlr4)" />
```

Comment:
💭 According to the linked docs, it sounds like these items are already included by NCrunch.

Reply:
Looks like NCrunch actually ignores items it doesn't recognize. I'll double-check tomorrow just to make sure.

Reply:
Actually I just tested it again: NCrunch won't copy the g4 files if they're only referenced by `Antlr4` items.

Reply:
Thanks, can you add a short comment here mentioning this?